### ISSUE TITLE
Replicate the supervised baseline

### ISSUE THREAD

**Issue by presencee5** [kzhou23/shape_pose_disent#2]
Hi,
Thank you for the inspiring work. I am trying to train a generative model in the supervised disentanglement setting and am wondering about the loss function.
According to the paper, the intermediate mesh "is replaced by the pseudo-groundtruth" in the supervised setting. So can we say the loss now becomes:
![loss](https://user-images.githubusercontent.com/18239152/106382039-ae766a00-63f7-11eb-876f-f9670eda279f.PNG)
Thanks in advance :-)

**Comment by kzhou23** [comment]
Thanks for your interest in our work:) By pseudo-groundtruth I mean the mesh generated from SMPL pose of X_1^s and SMPL shape of X^t. We have groundtruth SMPL shape and pose for every sequence in AMASS, so we utilize it to generate pseudo-groundtruth for pose transfer. You just need to replace the X_1^t in the second term by this pseudo-groundtruth to make it work..

**Comment by mikecheninoulu** [comment]
Hi, dear Zhou,

Thanks for your impressive work, I currently want to reproduce your work, but I cannot get the groundtruth SMPL shape and pose, especially from FAUST and DFAUST datasets. Could you indicate with more details how to get the groundtruth SMPL shape and pose?

**Comment by presencee5** [comment]
Hi,

Try AMASS dataset according to the paper.  AMASS unifies several human shape datasets including DFAUST. 
I guess AMASS reparameterizes these datasets by fitting them into the SMPL template. So getting SMPL templete and DFAUST subset of AMASS from their own websites may work.
